GodotAddinVS/GodotAddinVS.csproj

```diff
@@ -78,10 +78,10 @@
     <PackageReference Include="Clide" Version="4.1.1" ExcludeAssets="runtime" />
     <PackageReference Include="Clide.Windows" Version="4.1.1" ExcludeAssets="runtime" />
     <PackageReference Include="GodotTools.IdeMessaging" Version="1.1.1" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="16.0.205" ExcludeAssets="runtime">
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="16.10.31321.278" ExcludeAssets="runtime">
       <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.5.2044">
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.11.35">
```

Comment:
Are these two upgrades necessary? Won't this prevent its use on VS19 installs that are still on previous minor versions? Specially `16.11.35` is only 3 months old.

Reply:
I can reverse this. Although I thought up to date dependencies, which are building with VS19 are cool. Having a robust system is always better if it has little to none negative effect on the end user.

Reply:
Yes, but we should give some room for upgrading. I think any version beyond 6 months old is fine.